howDoesCryptyWork.html

```diff
@@ -34,7 +34,28 @@ <h1>How does the Caesar Cipher work?</h1>
       space or line break it is going to skip it and continue with the next
       character. This means every space and line break are going to result in a
       space or line break.
-      <h1>How does the ****** ****** work?</h1>
+
+      <h1>How does the Vigenère cipher work?</h1>
+      The Vigenère cipher does work by using a Keyword or Phrase as Key. if the Text to Encode is longer than the Key itself we multiply the key until Key & TextToEncode have the same length and convert it both to uppercase. Then each letter of the plaintext is shifted based on the corresponding letter in the Keyword. Crypty
```

Comment:
Typo: "if" instead of "If"
```suggestion
      The Vigenère cipher does work by using a Keyword or Phrase as Key. If the Text to Encode is longer than the Key itself we multiply the key until Key & TextToEncode have the same length and convert it both to uppercase. Then each letter of the plaintext is shifted based on the corresponding letter in the Keyword. Crypty
```